### PR TITLE
BAH 495: Scroll issues in Lab Investigation chart

### PR DESCRIPTION
**Pull request by shilpa-iplit**
Scroll issues in Lab Investigation chart

### CODE REVIEW

File: ui/app/styles/bahmni-components/_table.scss [hunk 1]
```diff
@@ -224,3 +224,32 @@ table.h-scroll {
 .table-scroll-x {
   overflow-x: scroll;
 }
+
```
Comment (sumanmaity112) [inline ui/app/styles/bahmni-components/_table.scss]: **Investigation chart** is very specific to **clinical module**. Shouldn't it (this style) be present in **ui/app/styles/clinical** folder?

Comment (shilpa-iplit) [reply ui/app/styles/bahmni-components/_table.scss]: @sumanmaity112  Can you please advise me in which .scss file these CSS changes should go?

Comment (shilpa-iplit) [reply ui/app/styles/bahmni-components/_table.scss]: @sumanmaity112 ok sure. Thank you.

**Comment by shilpa-iplit** [comment]
JIRA issue URL: https://bahmni.atlassian.net/browse/BAH-495

File: ui/app/styles/clinical/_program.scss [hunk 2]
```diff
@@ -254,4 +254,63 @@
     height: 120px;
     margin-bottom: 10px;
   }
-}
\ No newline at end of file
+}
+
+/*scroll investigation chart table*/
```
Comment (krishnanspace) [inline ui/app/styles/clinical/_program.scss]: Hi @shilpa-iplit ,I think you should remove the comment.Apart from this,everything looks fine

**Pull request closed without merging**